pkg/process/checks/pod.go

```diff
@@ -48,10 +49,15 @@ func (c *PodCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.MessageB
 		return nil, err
 	}
 
+	clusterID, err := clustername.GetClusterID()
+	if err != nil {
+		return nil, err
+	}
+
 	podList, err := kubeUtil.GetRawLocalPodList()
 	if err != nil {
 		return nil, err
 	}
 
-	return orchestrator.ProcessPodlist(podList, groupID, cfg, cfg.HostName, cfg.KubeClusterName)
+	return orchestrator.ProcessPodlist(podList, groupID, cfg, cfg.HostName, cfg.KubeClusterName, clusterID)
```

Comment:
You should also update https://github.com/DataDog/datadog-agent/blob/06053c724f8e4ef2a85c2ba667eafa3159a3e2ea/pkg/clusteragent/orchestrator/controller.go#L119

Reply:
good catch! Updated (added the header as well).